src/shims/ApiCompatBaseline.netcoreapp.netstandard20.txt

```diff
@@ -1 +1,17 @@
-Total Issues: 0
+Compat issues with assembly mscorlib:
+TypesMustExist : Type 'System.TupleExtensions' does not exist in the implementation but it does exist in the contract.
```

Comment:
I understand why they exist and I'm not particularly concerned. To fix them we need to start generating the shims based of NET47 instead of NET461. We can do that but it might be a larger undertaking then I wanted right now.